catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneDirectoryIndex.java

```diff
@@ -105,6 +105,13 @@ protected IndexSearcher createIndexSearcher(final IndexReader indexReader) {
     return Collections.emptyList();
   }
 
+  @Override
+  public List<String> getSuggestedNames(String queryString, int maxResults)
+      throws GeoEntryQueryException {
+    LOGGER.debug("getSuggestedNames is not implemented for GeoNamesQueryLuceneDirectoryIndex");
```

Comment:
Do we need to expose all these implementation details?  We could simply log `Not implemented`